java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture1/tasks/_7/Number7.java

```diff
@@ -0,0 +1,54 @@
+package com.zoolatech.lecture1.tasks._7;
+
+import java.util.Scanner;
+
+/**Write a program that prints all odd numbers from N to 1.
+ *  N should be read from the input.
+ *  Every number should be printed on the new line.
+ *  Create two versions of the program:
+ *  one should use the for loop, second - the while loop.
+ */
+
+public class Number7 {
+    public static void main(String[] args) {
+        Number7 number7 = new Number7();
+        number7.allOddNumbersForLoopVersion();
+        number7.allOddNumbersWhileLoopVersion();
+    }
+
+    private void allOddNumbersForLoopVersion() {
+        Scanner scanner = new Scanner(System.in);
+        int oddNumber;
+        System.out.println("Input number for-loop: ");
+        oddNumber = scanner.nextInt();
+
+        for (int i = 0; i <= oddNumber; i++) {
+            int number = oddNumber-i;
+
+            if (number % 2 != 0) {
+                System.out.println(number);
+            }
+            if (number % 2 == 0) {
+                number += 1;
```

Comment:
`number += 1;` actually doesn't do anything cause the variable will be redefined during the next iteration